core/src/components/item-divider/item-divider.scss

```diff
@@ -75,18 +75,7 @@
 .item-divider-inner {
   @include margin(0);
   @include padding(var(--inner-padding-top), null, var(--inner-padding-bottom), null);
-
-  /* stylelint-disable */
-  @include ltr() {
-    padding-right: calc(var(--ion-safe-area-right, 0px) + var(--inner-padding-end));
-    padding-left: var(--inner-padding-start);
-  }
-
-  @include rtl() {
-    padding-right: var(--inner-padding-start);
-    padding-left: calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-end));
-  }
-  /* stylelint-enable */
+  @include safe-area-padding(null, var(--inner-padding-end));
```

Comment:
This is no longer using the `--inner-padding-start` property:

<table>
<tr>
  <td>Before</td>
  <td>After</td>
</tr>
<tr>
<td>

```css
.item-divider-inner {
  padding-right: calc(var(--ion-safe-area-right, 0px) + var(--inner-padding-end));
  padding-left: var(--inner-padding-start);
}

:host-context([dir=rtl]) .item-divider-inner {
  padding-right: var(--inner-padding-start);
  padding-left: calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-end));
}

[dir=rtl] .item-divider-inner {
  padding-right: var(--inner-padding-start);
  padding-left: calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-end));
}

@supports selector(:dir(rtl)) {
  .item-divider-inner:dir(rtl) {
    padding-right: var(--inner-padding-start);
    padding-left: calc(var(--ion-safe-area-left, 0px) + var(--inner-padding-end));
  }
}
```

</td>
<td>

```css
.item-divider-inner {
  padding-right: calc(var(--ion-safe-area-right, 0) + var(--inner-padding-end));
}

:host-context([dir=rtl]) .item-divider-inner {
  padding-left: calc(var(--ion-safe-area-left, 0) + var(--inner-padding-end));
  padding-right: unset;
}

[dir=rtl] .item-divider-inner {
  padding-left: calc(var(--ion-safe-area-left, 0) + var(--inner-padding-end));
  padding-right: unset;
}

@supports selector(:dir(rtl)) {
  .item-divider-inner:dir(rtl) {
    padding-left: calc(var(--ion-safe-area-left, 0) + var(--inner-padding-end));
    padding-right: unset;
  }
}
```

</td>
</tr>
</table>